instana/instrumentation/asyncio.py

```diff
@@ -14,27 +14,33 @@
     @wrapt.patch_function_wrapper('asyncio','ensure_future')
     def ensure_future_with_instana(wrapped, instance, argv, kwargs):
         if config['asyncio_task_context_propagation']['enabled'] is False:
-            return wrapped(*argv, **kwargs)
+            from opentracing.scope_managers.contextvars import no_parent_scope
+            with no_parent_scope():
```

Comment:
Why the no_parent_scope? I think we would be very much interested in NOT breaking the span propagation.

Reply:
Yes, but this block runs only when I ask to break context propagation. 
I am using it to guarantee it will not be propagated; otherwise, this `if` block (and the whole function, IMHO) is useless.

Reply:
You are right, I missed the point. This is that underdocumented one of a kind mummy that the customer is supposed to use manually with `from instana.configurator import config`.  I wonder if there is even a single customer making use of this.